src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,29 @@
+/* Experimental text primitive.
+ */
+
+var getMeshMixin = require('../getMeshMixin');
```

Comment:
Yeah, the box example uses the material component so it benefits from the mixin, but we don't need it here.